packages/forms/test/template_integration_spec.ts

```diff
@@ -1511,8 +1511,314 @@ import {NgModelCustomComp, NgModelCustomWrapper} from './value_accessor_integrat
            expect(onNgModelChange).toHaveBeenCalledTimes(2);
            tick();
          }));
+
+      it('should validate max', fakeAsync(() => {
+           const fixture = initTest(NgModelMaxValidator);
+           fixture.componentInstance.max = 10;
+           fixture.detectChanges();
+           tick();
+
+           const max = fixture.debugElement.query(By.css('input'));
```

Comment:
Similar to a comment above, we can retrieve `nativeElement` property sooner (and also use `input` as an input element name):
```suggestion
           const input = fixture.debugElement.query(By.css('input')).nativeElement;
```